### DISCUSSION TITLE
Unable to load local raster

### DISCUSSION THREAD

**Discussion by franegrao** [giswqs/geebook#8] [Q&A] [answered]
I get a ModuleNotFoundError: No module named 'osgeo'
when running pg 87 (top) snippet for loading local raster.
I checked and pygis is installed in my active conda gee environment from which I initiated jupyter lab.
This is my error message. Apparently the error is generated when trying to import gdal from osgeo. 
![Screenshot 2024-01-30 065733](https://github.com/giswqs/geebook/assets/44303502/588fce43-e7ca-4139-89fd-5b94d94f3ede)
Any ideas as to what might be wrong?

**Comment by franegrao** [comment]
I created a second environment in which, apart from pygis (whose installation did not throw any errors) I installed
gdal, rasterio, localtileserver and it worked. 
Anyway, I believe this shouldn't have been necessary, as the technical requirements for the chapter mention only the installation of pygis.

**Comment by giswqs** [comment, marked as answer]
The recent GDAL package updates might have caused installation issues on Windows. Try installing it using the following command if `import gdal from osgeo` fails. 

```bash
pip install -U --find-links https://girder.github.io/large_image_wheels GDAL
```

**Reply by franegrao** [reply]
Hi,
Thank you for your answer.
Just to keep you posted:
- from osgeo import gdal does not work.
- I tried pasting:    _pip install -U --find-links https://girder.github.io/large_image_wheels GDAL_   in anaconda prompt for my original gee but got several errors:
- 
![image](https://github.com/giswqs/geebook/assets/44303502/7084ec12-1b83-44e0-ab59-675978e3818c)

This could apparently have to do with the version of Microsoft visual C++?? I am working on windows 11. 
I suppose pygis will be fixed sometime and, as said, expressly adding gdal, rasterio and localtileserver to the environment, appears to solve the problem for now.
